cluster/gce/gci/configure.sh

```diff
@@ -56,13 +56,15 @@ EOF
 }
 
 function validate-python {
-  local ver=$(python -c"import sys; print(sys.version_info.major)")
-  echo "python version: $ver"
-  if [[ $ver -ne 2 && $ver -ne 3 ]]; then
+  local ver=$(python3 -c"import sys; print(sys.version_info.major)")
```

Comment:
1. isn't python to be installed for ubuntu 20.04+ version 3?
2. besides this imperative installation, is it possible to leverage cloud-init way?

Reply:
Looks like new version of ubuntu build already add python installation. but anyway, we have conditions and if python already exist, will skip installation.